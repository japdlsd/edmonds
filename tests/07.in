30 150
1 3 15
1 4 16
1 5 31
1 6 1
1 7 32
1 8 32
1 14 15
1 15 12
1 18 25
1 19 33
1 21 25
1 26 29
1 27 34
1 29 36
2 8 26
2 11 10
2 12 29
2 14 2
2 26 6
2 28 26
3 11 20
3 12 20
3 19 38
3 22 40
3 26 41
3 27 20
3 29 10
4 5 44
4 6 28
4 7 32
4 9 21
4 11 21
4 14 8
4 16 32
4 19 24
4 25 16
4 26 17
4 27 36
4 28 14
5 6 31
5 18 35
5 21 17
5 26 11
6 9 30
6 10 42
6 13 4
6 16 38
6 24 14
6 26 48
6 29 8
7 10 24
7 11 49
7 12 26
7 16 1
7 20 22
7 22 16
8 13 43
8 14 17
8 15 5
8 17 20
8 25 42
8 26 31
9 11 12
9 12 37
9 13 14
9 15 33
9 16 7
9 18 19
9 20 40
9 23 48
9 26 4
9 28 49
9 29 4
9 30 50
10 11 33
10 13 5
10 14 19
10 15 40
10 16 17
10 18 24
10 19 25
10 22 20
11 16 11
11 17 13
11 18 16
11 19 14
11 20 16
11 21 23
11 22 37
11 23 50
11 26 18
11 28 22
11 29 41
12 13 14
12 22 4
13 15 50
13 24 25
13 30 12
14 15 20
14 21 19
14 25 10
14 26 30
14 27 32
15 16 43
15 18 12
15 24 28
15 27 28
15 28 11
16 17 37
16 18 33
16 22 42
16 23 11
16 26 35
16 27 20
16 28 41
16 30 41
17 22 45
18 22 3
18 24 21
18 26 26
19 20 50
19 21 31
19 22 3
19 27 43
19 28 44
20 21 40
20 23 21
20 24 50
20 25 24
20 27 37
20 28 30
20 30 15
21 23 50
21 25 40
21 26 26
21 28 14
21 30 5
22 23 11
22 25 1
23 24 22
23 30 16
24 25 13
24 27 12
24 29 27
25 26 39
26 27 45
26 28 7
27 29 23
28 30 43
29 30 31
