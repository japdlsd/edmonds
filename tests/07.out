143
24 27
22 25
21 30
20 23
15 18
11 19
10 13
9 29
8 17
7 16
5 26
4 28
3 12
2 14
1 6

